k8s/helm/testkube-api/templates/deployment.yaml

```
@@ -156,6 +156,10 @@ spec:
               value:  "{{ .Values.podStartTimeout }}"
             - name: CDEVENTS_TARGET
               value: "{{ .Values.cdeventsTarget }}"
+            {{- if not .Values.global.testWorkflows.createOfficialTemplate }}
```

```suggestion
            {{- if not .Values.global.testWorkflows.createOfficialTemplates }}
```

Our values.yaml var is :

global.testWorkflows.createOfficialTemplates